src/aics-image-viewer/components/GlobalVolumeControls.tsx

```diff
@@ -47,7 +42,7 @@ export default class GlobalVolumeControls extends React.Component<GlobalVolumeCo
     return newImage || newSliderValue || newPathTraceValue;
   }
 
-  createSliderRow = (label: string, start: number[], max: number, propKey: string) => (
+  createSliderRow = (label: string, start: number[], max: number, propKey: UserSelectionKey) => (
```

Comment:
how is UserSelectionKey different than ChannelStateKey?

Reply:
`type UserSelectionKey = keyof UserSelectionState` (vs. `ChannelStateKey = keyof ChannelState`). I'm not totally sure that these types should have aliases, since they only save a handful of characters each and might be less clear than writing out the full expression, but they are both common enough to merit shortcutting.

Reply:
my question really was leading to whether there's a way to merge the two concepts: are the keys of UserSelectionState really that different from those of ChannelState?

Reply:
The difference has to do not just with the type it keys into but which `App` methods these handlers call down to, e.g. `handleChangeUserSelection` for `UserSelectionKey` vs. `changeOneChannelSetting` for `ChannelStateKey`. The only `App` method that takes both as written is `handleChangeToImage`, which I left loosely typed because I couldn't get typing as strict as I wanted for complicated reasons of generics and type guards. (I'll get back to it for symbolic constants.)